massgov/mayflower-patternlab
# remove limiting width on subtitle

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Removes unneeded width limitation on page intro subtitle affecting the bio page.
## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8642)

## Steps to Test


1. View the bio page (http://localhost:3000/?p=pages-bio-page) at various widths (specifically tablet)
2. Ensure that the subtitle ("Example Job Title")does not break early

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
